draft-ietf-mls-protocol.md

```diff
@@ -2729,6 +2729,12 @@ itself) and S the other child:
 * D is in the resolution of C, and the intersection of P's `unmerged_leaves`
   with the subtree under C is equal to the resolution of C with D removed.
 
+* If D is a parent node, the `unmerged_leaves` field of D is a subset of the
+  `unmerged_leaves` field of P.
+
+* If D is a leaf node, then the leaf index of D does not appear in the
+  `unmerged_leaves` field of P or any of its ancestors.
```

Comment:
This is almost implied by the resolution equation for the same reason as the new first condition.
Almost because we can't prove that `D` is not in the unmerged leaves of nodes not authenticated by `D`.